maps/Sample_Frugal.html

```diff
@@ -17,6 +17,9 @@
         model.mapmodel.addTile(12,12,1,1,'None');
         model.mapmodel.addTile(13,12,0,1,'Tree');
         model.mapmodel.addTile(14,12,0,2,'None');
```

Comment:
A change I would like to see here is an update to `default_map.map` and then use `loadMap.py` to generate and update this file rather than editing it directly.

Reply:
OK. I did another commit there the diamonds are in default.map. I regenerated Sample_Frugal.html from that default.map file. You're right that default.map is a better place to store this, since Sample_Frugal is temporary and likely to be overwritten.